config/line_sensor_param.yaml

```
@@ -0,0 +1,2 @@
+black_line: 230
+white_line: 700
```

```suggestion
black_line: 273
white_line: 771
```

Já da pra colocar os valores que atuais que estão na PR do modelo